src/models/commands/update-one-changeset-sql-command-attributes.model.ts

```diff
@@ -0,0 +1,18 @@
+export interface UpdateOneChangesetSQLCommandAttributes {
+  /** Fully-qualified class which specifies a ChangeExecListener */
+  changeExecListenerClass?: string | undefined;
```

Comment:
**nitpick (non-blocking):** Very minor nit; in TS using the `?` character after a token but before the `:` character tells the compiler that the property being described is of the type declared OR undefined. 

So, technically we can rewrite all of these instances as :

```suggestion
  changeExecListenerClass?: string;
```

Very minor, and it wont affect any of the runtime or compile time developer experience. Just sharing a perspective; no action required 🚀

Reply:
Nice to know it! I guess it was because of the autogenerated protobuf TS code.

Reply:
I changed it for string, number and boolean